vendor/document-title/document-title.js

```diff
@@ -11,7 +11,14 @@ var routeProps = {
   // that will be the document title. The `collectTitleTokens` action
   // stops bubbling once a route is encountered that has a `title`
   // defined.
-  title: null
+  title: null,
+
+  // 'volatileController' must be used instead of `controller` when creating
+  // a computed property for `title` or `titleToken` because `controller`
+  // is not observable.
+  volatileController: Ember.computed(function() {
+    return this.get('controller');
+  }).volatile()
```

Comment:
This should probably not be included in the addon...it's an implementation detail of the consuming application, and it's not the only way a token might change.  I don't think the addon itself should provide this.